chaos_wrapper.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+function rollback() {
+    git checkout master
+    git checkout HEAD^
```

Comment:
That doesn't push the changes or anything, the actual fix is left up to us. This just gives us the opportunity to make it.